myuw-ios/WebViewController.swift

```diff
@@ -150,6 +187,70 @@ class WebViewController: UIViewController, WKNavigationDelegate {
     
     // webview policy response handler
     func webView(_ webView: WKWebView, decidePolicyFor navigationResponse: WKNavigationResponse, decisionHandler: @escaping (WKNavigationResponsePolicy) -> Void) {
+        
+        if let response = navigationResponse.response as? HTTPURLResponse {
+            if response.statusCode == 500 {
```

Comment:
Reminder that we want to add some buisiness logic here around 401's and other non 200 responses.